Makefile

```diff
@@ -92,22 +92,24 @@ fix-cs: fix-cs-php
 test: test-cs test-advanced test-unit
 
 test-cs:
-	bin/tools vendor/bin/php-cs-fixer fix --allow-risky yes --dry-run --verbose --diff
-	bin/tools bin/console --env=test lint:twig templates
+	bin/tools bin/php-cs-fixer fix --allow-risky yes --dry-run --verbose --diff
+	bin/tools bin/phpcs
+	bin/tools bin/console lint:twig templates --env=test
+	bin/tools php bin/console lint:yaml config --parse-tags --env=test
 	bin/node-tools npm run lint
 	bin/node-tools npm run lint:css
 
 test-advanced:
 	bin/tools sh -c "APP_DEBUG=1 APP_ENV=test bin/console c:w"
-	bin/tools vendor/bin/phpstan analyse --configuration phpstan.neon --level max --no-progress .
+	bin/tools bin/phpstan.phar analyse --configuration phpstan.neon --level max --no-progress .
```

Comment:
why do you prefer the phar ?

Reply:
because it's budled, it doesn't have any depandancy. but in this particular situation, it doesn't make any difference because it's a dev requirement.